scripts/Makefile.docker

```diff
@@ -25,12 +25,18 @@ NO_XSDEV_IMAGE := $(if $(HAVE_XSDEV_IMAGE),,1)
 # check if fd0 refers to tty
 HAVE_TTY = $(shell [ -t 0 ] && echo y)
 
+# helper to escape MAKEOVERRIDES
+space_escape := _-_SPACE_-_
+define escape_overrides
+$(subst $(space_escape),\ ,$(addprefix -e ,$(subst \ ,$(space_escape),$(MAKEOVERRIDES))))
```

Comment:
Fancy makefile. I don't understand what's going on based on my poor knowledge on Makefile. But if it works, it looks good.

Reply:
it was fix for following cmds:
```
make verilog ... PLDM_ARGS="--difftest-config H" ...
```

the `MAKEOVERRIDES` was expended as `PLDM_ARGS=--difftest-config\ H`,
and the original `$(addprefix -e , ...)` will convert it to `-e PLDM_ARGS=--difftest-config\ -e H`.
but what we need was `-e PLDM_ARGS=--difftest-config\ H`.

This commit fix it by first replace all space`\ ` to placeholder `_-_SPACE_-_` before `$(addprefix ...)`,
and then replace placeholder back to space`\ `.